src/main/java/nextstep/subway/line/ui/LineController.java

```diff
@@ -31,6 +31,14 @@ public ResponseEntity<List<LineResponse>> findAllLines() {
         return ResponseEntity.ok(lineService.findLineResponses());
     }
 
+    @GetMapping("pages")
```

Comment:
```suggestion
    @GetMapping("/pages")
```

다른 메서드들과 동일하게 /를 붙여보면 좋겠네요~